tools/js-sdk-release-tools/src/hlc/utils/modifyChangelogFileAndBumpVersion.ts

```diff
@@ -18,6 +18,8 @@ export function makeChangesForFirstRelease(packageFolderPath: string, isStableRe
     
 ## ${newVersion} (${date})
 
+### Features Added
```

Comment:
Could we also update this one: https://github.com/Azure/azure-sdk-tools/blob/main/tools/js-sdk-release-tools/src/hlc/utils/modifyChangelogFileAndBumpVersion.ts#L30-L40?

Reply:
Nice catch
Added